pkg/projects/state.go

```diff
@@ -182,6 +184,29 @@ func (state *state) GetDm() *metrics.DeviceMetric {
 	return state.deviceInfo.deviceMetric
 }
 
+//GetEVEIPs returns EVE IPs from info
+func (state *state) GetEVEIPs() (ips []string) {
+	if state.device.GetDevModel() == defaults.DefaultRPIModel {
+		dInfo := state.GetDinfo()
+		if dInfo != nil {
```

Comment:
I wonder if it would be better to introduce generic wrapper methods for this (if X !=nil ... pattern) to access various bits of state.